python/physicsobjects/DiLepton.py

```diff
@@ -13,7 +12,9 @@ class DiLepton(object):
     '''
     '''
     def __init__(self, pair, sv, pv, bs):
-        self._leptons = sorted(pair, key = lambda x : x.pt(), reverse = True)
+        # self._leptons = sorted(pair, key = lambda lep : (-abs(lep.pdgId()),lep.pt()), reverse = True)
+        self._leptons = pair
```

Comment:
why did you remove the pt sorting?

Reply:
ok, I'm confused, there are several places where these leptons can be sorted, either by pt or by flavour (i.e. if you have one electron and a one muon, the electron always comes first regardless of its pt).
Can you explain the logic again and make sure it's not inconsistent?